data-manager/jdbc/src/test/java/org/orbisgis/orbisdata/datamanager/jdbc/JdbcDataSourceTest.java

```diff
@@ -772,17 +782,15 @@ void testGetLocation() {
     void testGetTableNames() {
         Collection<String> names = ds1.getTableNames();
         assertNotNull(names);
-        assertEquals(38, names.size());
```

Comment:
Why are you removing assertion ?

Reply:
Because this is not good to test if all tables exist (user+ system tables). The number of system tables is not stable and can change